ChatClient/ChatClient/LoginPage.xaml.cs

```diff
@@ -17,7 +17,7 @@ namespace ChatClient
 	public partial class LoginPage : ContentPage
 	{
 		private const int defaultPort = 5318;
-		private const string hostname = "10.0.2.2";
+		private const string hostname = "192.168.2.12";
```

Comment:
This has nothing to do with ActivityIndicator -> should have been separate commit